PostGreKeyRotation/Deployment/helm/helmgensecret.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+kubectl create secret docker-registry acrsecret --docker-server jwrotateacr.azurecr.io --docker-email juswen@microsoft.com --docker-username jwrotateacr --docker-password <acr-password> --dry-run=true --output yaml
```

Comment:
Should we sanitize the --docker-server value, email and username?

Reply:
Ok fixed this.